packaging/linux/post_install.sh

```diff
@@ -39,10 +39,12 @@ if [ -z "$currlink" ] ; then
         if killall kbfsfuse &> /dev/null ; then
             echo Shutting down kbfsfuse...
         fi
-        rmdir "$rootlink"
         echo You must run run_keybase to restore file system access.
     fi
-    if ln -s "$mount1" "$rootlink" &> /dev/null ; then
+    if rmdir "$rootlink" &> /dev/null ; then
+        echo Replacing old $rootlink directory.
+    fi
```

Comment:
Maybe put all of this inside an extra check like:

```
if [ -d "$rootlink" ] ; then ...
```

That way if the `rmdir` fails, we know it's unexpected (and not just the case where we put a symlink there last time), and we can at least print something informative. Your call on whether it's worth trying to fail the install -- I'd probably need to test that on a couple different distros and see if it does something helpful.

Reply:
Did that, thanks.  Tested in both the empty and non-empty cases, seems to work fine.  Just going to print a warning in the non-empty case, rather than failing the package upgrade.  We can revisit later if needed.